app/main.py

```diff
@@ -2,0 +1,18 @@
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+
+    if len(command_parts) != 3 or command_parts[0] != "cp":
+        raise ValueError("Invalid command format. "
+                         "Use 'cp <source_file> <destination_file>'.")
+
+    source_file, destination_file = command_parts[1], command_parts[2]
```

Comment:
Code Style: You are using descriptive variable names like `source_file` and `destination_file`, which is great. This makes your code easier to read and understand.